tests/Unit/RpcLoggerTest.php

```
@@ -354,10 +356,358 @@
+                ];
+            }
+        };
+
+        // Custom processor for credit card numbers (mock)
+        $ccProcessor = new class implements ContextProcessorInterface {
+            public function canProcess(mixed $value): bool
+            {
+                return \is_string($value) && \preg_match('/^\d{4}-?\d{4}-?\d{4}-?\d{4}$/', $value);
+            }
+
+            public function process(mixed $value, callable $recursiveProcessor): mixed
```

PHPMD (2.15.0)

Avoid unused parameters such as '$recursiveProcessor'. (Unused Code Rules)

(UnusedFormalParameter)